internal/server/v1/propertyvalues/golden/bulk_property_values_in/containedIn1.json

```diff
@@ -22556,31 +22556,31 @@
             "PowerPlant"
           ],
           "dcid": "dc/kg6g6mkqg9j2g",
-          "provenanceId": "dc/df6wc4"
+          "provenanceId": "dc/kl1tbs3"
```

Comment:
the prov is supposed to be stable

Reply:
This is expected because the provenance actually changed cl/487410641